src/cvodes/cvodes_impl.h

```diff
@@ -1183,6 +1185,14 @@ int cvSensRhs1InternalDQ(int Ns, sunrealtype t, N_Vector y, N_Vector ydot,
                          int is, N_Vector yS, N_Vector ySdot, void* fS_data,
                          N_Vector tempv, N_Vector ftemp);
 
+/* Utility functions to tell CVODE to free the user data.
+   This is used by the Python interfaces. */
+
+int CVodeSetOwnUserData(void* cvode_mem, sunbooleantype own_user_data);
+
+int CVodeSetOwnUserDataB(void* cvode_mem, int which,
+                         sunbooleantype own_user_data);
```

Comment:
Prefix internal functions with `cv`
```suggestion
int cvSetOwnUserData(void* cvode_mem, sunbooleantype own_user_data);

int cvSetOwnUserDataB(void* cvode_mem, int which, sunbooleantype own_user_data);
```